java/common/src/main/java/io/strimzi/common/TracingInitializer.java

```diff
@@ -9,14 +9,26 @@
 import io.opentracing.util.GlobalTracer;
 import io.jaegertracing.Configuration;
 
+/**
+ * Provides methods for configuring tracing
+ */
 public class TracingInitializer {
-
+    /**
+     Initializes Jaeger Tracing by loading the configuration from the environment
+     and registering the tracer as a global tracer
+     @return the initialized Jaeger Tracer object
+     */
```

Comment:
If you add empty line between the description and the return tag it will be much more readable. ormally you would also have the `*` all along. E.g.:

```
    /**
     * Initializes Jaeger Tracing by loading the configuration from the environment
     * and registering the tracer as a global tracer
     *
     * @param xxx     Some description
     *  
     * @return the initialized Jaeger Tracer object
     */
```

When it is only return without params, you can also do somehting like this:

```
    /**
     * @return    The whole description
     */
```

Reply:
@scholzj Is this better? Still a little unsure of how to format these properly.